### PR TITLE
Pipe in LDFLAGS env var to enable fully static bin

### PR DESCRIPTION
**Pull request by agouin**
Fixes https://github.com/cosmos/relayer/actions/runs/3587267089/jobs/6037415594

### CODE REVIEW

**Review by boojamya — APPROVED** [review]
🧙‍♂️

File: Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ RUN if [ -d "/go/bin/linux_${TARGETARCH}" ]; then mv /go/bin/linux_${TARGETARCH}
 
 # Use minimal busybox from infra-toolkit image for final scratch image
 FROM ghcr.io/strangelove-ventures/infra-toolkit:v0.0.6 AS busybox-min
-RUN addgroup --gid 1025 -S relayer && adduser --uid 1025 -S relayer -G relayer
+RUN addgroup --gid 1000 -S relayer && adduser --uid 100 -S relayer -G relayer
```
Comment (joeabbey) [inline Dockerfile:27]: did you want the UID of 100 or 1000?

File: Dockerfile [hunk 2]
```diff
@@ -64,7 +64,7 @@ COPY --from=busybox-min /etc/ssl/cert.pem /etc/ssl/cert.pem
 
 # Install relayer user
 COPY --from=busybox-min /etc/passwd /etc/passwd
-COPY --from=busybox-min --chown=1025:1025 /home/relayer /home/relayer
+COPY --from=busybox-min --chown=100:1000 /home/relayer /home/relayer
```
Comment (joeabbey) [inline Dockerfile:67]: Looks like you wanted the UID of 100

Comment (agouin) [reply Dockerfile:67]: yep! 100:1000 is what it was in the previous rly images

**Pull request merged**